ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -11,7 +11,9 @@
       },
       {
         "UserRoles": [
-          "caseworker-civil-admin"
+          "caseworker-civil-admin",
+          "judge-profile",
```

Comment:
@kurt-uk are these not access profiles?
Im not sure if this will cause issues with uploading on master. Do we know if these roles have been created on the ccd admin web portal

Reply:
These seem to be pre-existing in the role to access profile base file - which is currently being uploaded. No issues!